### PR TITLE
Help button

### PR DESCRIPTION
**Pull request by kesharwaniyanshi**
## Related Issue

Closes: #1582 

## Description

Changed the location of Help button and placed it along the Submit button in the Form which looks better than the previous location oh help. 
Shifted the Help button in the same div as Submit button and made some changes in css of help button. 

## Screenshots

image

## Checklist 

- [x] My code adheres to the established style guidelines of the project.
- [x] I have included comments in areas that may be difficult to understand.
- [x] My changes have not introduced any new warnings.
- [x] I have conducted a self-review of my code.

### CODE REVIEW

**Comment by kesharwaniyanshi** [comment]
Please review the PR and merge it.

File: reportstary.css
Comment (utk2103) [inline reportstary.css:1]: Remove unnecessary comments

**Review by akshitagupta15june — APPROVED** [review]

**Comment by akshitagupta15june** [comment]
Please do a fetch and merge to update with current codebase! and resolve conflicts

**Comment by kesharwaniyanshi** [comment]
Resolved the conflicts.

**Pull request merged**